File: C_code/assign1.c
#include <stdio.h>

int main()
{
    int totinvest =22500;//Total investment
    //value of each share 
    int value = 50;
    //discount on each share 
    int discount = ((double)10/100)*value;
    //Market price
    int mprice =value - discount;
    //total no.of shares purchased 
    int no_shares = totinvest/mprice;
    //total dividend
    int dividend = (double)12/100*value*no_shares;
    //rate of return 
    double retrnpercent = (double)dividend/totinvest*100;
    printf("No.of shares purchased = %d\n",no_shares);
    printf("Total dividend = %d\n",dividend);
    printf("Rate of return(approx to nearest integer) = %d\n",(int)retrnpercent);
    if (no_shares == 500 && dividend == 3000 && (int)retrnpercent == 13) 
    printf("Verified\n");
    else 
    printf("There is an error\n");
    return 0;
}

File: py_code/assign1.py
# Total investment
totinvest =22500
# value of each share 
value = 50
#discount on each share 
discount = (float(10)/100)*value
#Market price
mprice =value - discount
#total no.of shares purchased 
no_shares = totinvest/mprice
#total dividend
dividend = float(12)/100*value*no_shares
#rate of return 
retrnpercent = float(dividend)/totinvest*100
print("No.of shares purchased = ",no_shares,)
print("Total dividend = ",dividend,)
print("Rate of return(approx to nearest integer) = ",int(retrnpercent),)
if (no_shares==500 , dividend==3000 , int(retrnpercent) == 13):
    print("Verified")
else :
    print("There is an error")	
